src/renderer/components/+nodes/route.tsx

```diff
@@ -124,7 +125,7 @@ export class NodesRoute extends React.Component {
     return this.renderUsage({
       node,
       title: "Memory",
-      metricNames: ["workloadMemoryUsage", "memoryAllocatableCapacity"],
+      metricNames: ["memoryUsage", "memoryAllocatableCapacity"],
```

Comment:
> No it is not the exact same

Need to clarify this. If it not the same then numbers would be different. But example here https://github.com/lensapp/lens/issues/5457 shows that only suffix should be changed. How we can be 100% sure that changing metric actually fixing the issue?

Reply:
It might be in his case that it is exactly 8x smaller (modulo rounding) but in my testing it wasn't exactly that ratio. However what I did find is that this is our only use of `workloadMemoryUsage` whereas on the cluster overview we use `memoryUsage`

Reply:
Maybe @nevalla remembers the difference (I am not an expert on Prometheus).